libsol/stake_instruction.c

```diff
@@ -367,14 +367,14 @@ static int print_stake_authorize_info(
     SummaryItem* item;
 
     item = transaction_summary_primary_item();
-    summary_item_set_pubkey(item, "Set stake auth.", info->account);
+    summary_item_set_pubkey(item, "Set stake auth", info->account);
 
     switch (info->authorize) {
         case StakeAuthorizeStaker:
-            new_authority_title = "New stake auth.";
+            new_authority_title = "New stake auth";
             break;
         case StakeAuthorizeWithdrawer:
-            new_authority_title = "New w/d auth.";
+            new_authority_title = "New w/d auth";
```

Comment:
On 503, similar screen reads "New withdraw auth". Not sure if 17 chars works there for some reason, but even if so, recommend that you change that one to be consistent.

Reply:
Looks like "withdraw" _just_ works here after removing the periods from "auth". It was butted up against the navigation arrows on both sides with.

FYI: 16 came from a full line of worst-case bold chars. "W" as per Ledger's page breaking logic.  We can get away with more in most cases